middleware/validate.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const templateDataDiff = require('../lib').templateDataDiff;
+const debug = require('debug')('middleware:validate');
+
+module.exports = (req, res, next) => {
+  const template = req.body.template;
+  debug('Validating template: %s', template);
+
+  if (!template) {
+    next({code: 'NoTemplate', message: 'Template must be a string'});
```

Comment:
is it 🤔 ? I was thinking that perhaps it would be useful to tell the user what they must do to fix it, i.e., pass a String template